superflore/utils.py

```diff
@@ -219,6 +213,18 @@ def resolve_dep(pkg, os):
         raise UnknownPlatform(msg)
 
 
+def get_distros():
+    index = get_cached_index()
+    return index.distributions
+
+
+def get_distros_by_status(status='active'):
+    distros = get_distros()
+    distros_by_status = [d for d in distros.keys(
+    ) if distros[d].get('distribution_status') == status]
+    return distros_by_status
```

Comment:
If superflore is python3 only this could be simplified to

```python
    return [d for d, info in get_distros().items()
            if info['distribution_status'] == status]

Reply:
Done, thanks!